stylesheets/commons/Brackets.less

```diff
@@ -550,6 +511,45 @@
 	}
 }
 
+.brkts-match-info-popup {
+	border-radius: 0.5rem;
+	box-shadow: 0 0.0625rem 0.25rem 0 rgba( 0, 0, 0, 0.12 );
+	position: absolute;
+	z-index: 19;
+
+	@media ( max-width: 768px ) {
+		position: fixed;
+		bottom: unset;
+		left: 1rem !important;
+		right: 1rem;
+		top: 50% !important;
+		max-height: 50vh;
+		width: auto !important;
+		transform: translateY( -50% );
+	}
+}
+
+@media ( min-width: 769px ) {
+	.brkts-match-info-popup-sc-ffa:not( .brkts-match-info-flat ) {
```

Comment:
wasn't sc-ffa getting nuked?

Reply:
not before i opened this PR
and git now does some BS...